### DISCUSSION TITLE
Add datepicker control to forms

### DISCUSSION THREAD

**Discussion by taubmann** [espanso/espanso#1224] [Feature requests and Ideas]
Hi there, for due dates and other date related inputs it would be great to have a special kind of the Form Element showing a date picker as a pop up to return definable date strings

**Comment by federico-terzi** [comment]
Hey @taubmann,

Thank you for the feedback! This is a duplicate of https://github.com/federico-terzi/modulo/issues/14, but given that modulo is being moved into the espanso directory in the next release, I'll keep this one open and close the other one.

Cheers :)
